src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private Long userId;
```

Comment:
Aren't 'id' and 'userId' the same?

Reply:
Yes, but I think better pass difference id variable for user and shopping cart.